### PR TITLE
MP-2467 Filter products by merchant in backoffice

### PR DESCRIPTION
**Pull request by az-joss**
## PR Description

- Ticket: https://spryker.atlassian.net/browse/MP-2467

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-feature-integration.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Install the required modules:
 ```bash
 composer require spryker-feature/marketplace-product: "dev-master" --update-with-dependencies
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-feature-integration.md:29]: please use release tag

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-feature-integration.md [hunk 2]
```diff
@@ -126,7 +127,11 @@ Enable the following behaviors by registering the plugins:
 | MerchantProductAbstractMapExpanderPlugin                     | Adds merchant names to product abstract search data.         |           | Spryker\Zed\MerchantProductSearch\Communication\Plugin\ProductPageSearch |
 | MerchantProductPageDataExpanderPlugin                        | Expands the provided ProductAbstractPageSearch transfer object's data by merchant names. |           | Spryker\Zed\MerchantProductSearch\Communication\Plugin\ProductPageSearch |
 | MerchantProductPageDataLoaderPlugin                          | Expands ProductPageLoadTransfer object with merchant data.   |           | Spryker\Zed\MerchantProductSearch\Communication\Plugin\ProductPageSearch |
+| Merchant\MerchantProductSearchWritePublisherPlugin           | Publish the product by merchant ids in ES. |           | Spryker\Zed\MerchantProductSearch\Communication\Plugin\Publisher |
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-feature-integration.md:130]:
```suggestion
| Merchant\MerchantProductSearchWritePublisherPlugin           | Publishes the product by merchant ids to ES. |           | Spryker\Zed\MerchantProductSearch\Communication\Plugin\Publisher |
```
please check everywhere

**Review by ilyakubanov — COMMENTED** [review]
Please merge master

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-feature-integration.md (hunk 2 above)
Comment (az-joss) [reply docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-feature-integration.md:130]: fixed

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-feature-integration.md (hunk 1 above)
Comment (az-joss) [reply docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-feature-integration.md:29]: command that install feature should be with `dev-master` according to "Feature integration guides"

**Review by ilyakubanov — DISMISSED** [review]

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**